doc/classes/Vector2.xml

```diff
@@ -321,9 +322,10 @@
 		</method>
 		<method name="reflect" qualifiers="const">
 			<return type="Vector2" />
-			<param index="0" name="n" type="Vector2" />
+			<param index="0" name="line" type="Vector2" />
 			<description>
-				Returns the result of reflecting the vector from a line defined by the given direction vector [param n].
+				Returns the result of reflecting the vector from a line defined by the given direction vector [param line].
+				[b]Note:[/b] [method reflect] differs from what other engines and frameworks call [code]reflect()[/code]. In other engines [code]reflect()[/code] takes a normal direction which is a direction perpendicular to the line. In Godot, you specify the direction of the line directly. Also see [method bounce] which does what most engines call [code]reflect()[/code].
```

Comment:
```suggestion
				[b]Note:[/b] [method reflect] differs from what other engines and frameworks call [code]reflect()[/code]. In other engines, [code]reflect()[/code] takes a normal direction which is a direction perpendicular to the line. In Godot, you specify the direction of the line directly. See also [method bounce] which does what most engines call [code]reflect()[/code].
```

Reply:
Amended the commit myself with those fixes.